sycl/include/sycl/ext/oneapi/bfloat16.hpp

```diff
@@ -32,6 +32,27 @@ Bfloat16StorageT bfloat16ToBits(const bfloat16 &Value);
 bfloat16 bitsToBfloat16(const Bfloat16StorageT Value);
 } // namespace detail
 
+static detail::Bfloat16StorageT from_float_fallback(const float &a) {
```

Comment:
This function is not part of the documented API, so it should not be in the public namespace `sycl::ext::oneapi`.  Can't it just be a private member function of the `bfloat16` class?

Reply:
Hi, @gmlueck 
Make sense, already done.